src/components/BookStatusPanel/BookStatus/BookStatus.module.css

```diff
@@ -0,0 +1,65 @@
+.green {
```

Comment:
For this case when we have a lot of repeating styles for all the statuses(everything the same except color) we can use [classnames](https://www.npmjs.com/package/classnames) to join the classes together.

After installing it,  you can do something like this:

- In css file:
```
.status {
 margins, padding, font,... all styling that 
}

.green {
styling just for color
}
.... for other colors
```

- in tsx file:
```
import cx from 'classnames'

const BookStatusRecord: Record<Status, string> = {
  [Status.AVAILABLE]: cx(styles.status, styles.green), 
  .....
}
```

This would merge both styles together, with a rule that whatever is the last function prop overrides previous props: 
for example if you put `padding: 2px` in styles.status, and `padding: 3px` in styles.green the end result of cx function would be` padding:3px`